src/interface.jl

```diff
@@ -254,24 +272,26 @@ function forward(cb::Composed, x)
     for t in cb.ts[2:end]
         res = forward(t, rv)
         rv = res.rv
-        logjac = res.logabsdetjac .+ logjac
+        logjac = res.logabsdetjac + logjac
     end
     return (rv=rv, logabsdetjac=logjac)
 end
 
 -###########
 -# Stacked #
 -###########
+const ZeroOrOneDimBijector = Union{Bijector{0}, Bijector{1}}
+
 """
     Stacked(bs)
     Stacked(bs, ranges)
-    stack(bs::Bijector...)
+    stack(bs::Bijector{Dim=0}...)
```

Comment:
`Dim=0`?

Reply:
What's the question? :smile: You think it should be `D=0` instead? Or the fact that we're restricting to only allow `D=0`?